homeassistant/components/vacuum/neato.py

```diff
@@ -188,6 +188,8 @@ def pause(self):
 
     def return_to_base(self, **kwargs):
         """Set the vacuum cleaner to return to the dock."""
+        if self._clean_state == STATE_CLEANING:
+            self.robot.pause_cleaning()
         self._clean_state = STATE_RETURNING
         self.robot.send_to_base()
```

Comment:
Forgot to change this to self.robot. return_to_base()

Reply:
https://github.com/stianaske/pybotvac/blob/master/pybotvac/robot.py#L118

You sure about that?

Reply:
I can check it about an hour but pretty sure send_to_base was changed to return_to_base recently. Searching this github for "send_to_base" shows this as its only usage.

Reply:
Nope, lol. Just didn't see that function defined in the code base. I don't know python though. I'll check with my neato in an hour or so